File: csharp/Board.cs
namespace GameOfLife {
    internal class Board {
        public IList<bool> cells { get; }
        public int height { get; }
        public int width { get; }

        public Board(int height, int width) {
            this.cells = new List<bool>();
            this.height = height;
            this.width = width;

            Random random = new();
            for (var i = 0; i < this.height * this.width; i++)
            {
                cells.Add(random.Next(2) == 1);
            }
        }

        public Board(Board other) {
            this.cells = new List<bool>(other.cells);
            this.height = other.height;
            this.width = other.width;
        }

        public void Draw() {
            for (var i = 0; i < this.height; i++) {
                for (var j = 0; j < this.width; j++) {
                    if (this.cells[(i * this.width) + j]) {
                        Console.Write("██");
                    }
                    else {
                        Console.Write("  ");
                    }
                }
                Console.WriteLine();
            }
        }

        public bool GetCell(int row, int col)
        {
            if (row >= 0 && row < this.height && col >= 0 && col < this.width)
            {
                return this.cells[(row * this.width) + col];
            }
            else
            {
                return false;
            }
        }
    }
}

File: vb/Board.vb
Class Board
    Public m_Cells As List(Of Boolean)
    Public m_Height As Integer
    Public m_Width As Integer

    Public Sub New(ByVal height As Integer, ByVal width As Integer)
        m_Height = height
        m_Width = width
        m_Cells = New List(Of Boolean)

        Dim random As Random = New Random
        For i = 1 To (height*width)
            m_Cells.Add(random.Next(2))
        Next i
    End Sub

    Public Sub New(ByRef other As Board)
        m_Cells = other.m_Cells.ToList()
        m_Height = other.m_Height
        m_Width = other.m_Width
    End Sub

    Public Sub Draw()
        For i = 0 To (m_Height-1)
            For j = 0 To (m_Width-1)
                If m_Cells((i * m_Width) + j) Then
                    Console.Write("██")
                Else
                    Console.Write("  ")
                End If
            Next j
            Console.WriteLine()
        Next i
    End Sub

    Public Function GetCell(ByVal row As Integer, ByVal col As Integer) As Boolean
        If (row >= 0) And (row < m_Height) And (col >= 0) And (col < m_Width) Then
            Return m_Cells((row * m_Width) + col)
        Else
            Return False
        End If
    End Function
End Class